recipes.mdwn

```diff
@@ -37,6 +37,7 @@ to improve your Awesome setup.
 * [Collision geometric navigation keybindings](https://github.com/Elv13/collision)
 * [Tyrannical dynamic tag managment framework](https://github.com/Elv13/tyrannical)
 * [Repetitive dynamic keybindings and macros](https://github.com/Elv13/repetitive)
+* [Alternative tag management for multiple screens](https://github.com/dakling/awesome-sharedtags)
```

Comment:
Either
- This links to GitHub and the previous commits that add (basically) a local copy of the GitHub contents should go.
- This links to `recipes/sharedtags-greedyview` instead.

Right now, the content added in the previous commits is not linked from anywhere and so could as well be removed.